library/Notifications/Widget/MemberSuggestions.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/* Icinga Notifications Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Notifications\Widget;
+
+use Icinga\Module\Notifications\Common\Database;
+use Icinga\Module\Notifications\Model\Contact;
+use ipl\Html\Attributes;
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\Stdlib\Filter;
+use Psr\Http\Message\ServerRequestInterface;
+
+class MemberSuggestions extends BaseHtmlElement
+{
+    protected $tag = 'ul';
+
+    /** @var string */
+    protected $searchTerm;
+
+    /** @var string */
+    protected $originalValue;
+
+    /** @var string[] */
+    protected $excludeTerms = [];
+
+    public function setSearchTerm(string $term): self
```

Comment:
This method is not used outside the class. Is there a reason, why this method is public?